app/main.py

```diff
@@ -1,6 +1,14 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    result = {}
+
+    def wrapper(*args) -> Any:
```

Comment:
```suggestion
    def wrapper(*args, **kwargs) -> Any:
```
Please note that the function can also accept key arguments

Reply:
If it's not correct, could you please provide more details, as I don't understand